.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: java
 jdk:
   - openjdk11
+  - openjdk14
```

Comment:
Do you need it? Did you know why you have added these config properties?

Reply:
It added because I pull updates from the remote repo. And I don't know if I need to commit it.